Netflix/Views/Home/HomeLatestMovieUIView.swift

```diff
@@ -0,0 +1,119 @@
+import UIKit
+import SnapKit
+
+class HomeLatestMovieUIView: UIView {
+    private let movieImage: UIImageView = {
+        let image = UIImageView()
+        image.image = Asset.latestMovieNetflix.image
+        image.contentMode = .scaleAspectFit
+        return image
+    }()
+    private let filmName: UILabel = {
+        let filmNameText = UILabel()
+        filmNameText.text = "Never Have I Ever"
+        filmNameText.numberOfLines = 2
+        filmNameText.font = UIFont.systemFont(ofSize: 50, weight: .bold)
+        filmNameText.textAlignment = .center
+        filmNameText.textColor = .white
+        return filmNameText
+    }()
+    private let hashtags: UILabel = {
+        let hashtagsText = UILabel()
+        hashtagsText.text = "Quirky • Youth • Teen • Comedy • Drama • Gal Pals"
+        hashtagsText.font = UIFont.systemFont(ofSize: 12)
+        hashtagsText.textAlignment = .center
+        hashtagsText.textColor = .white
+        return hashtagsText
+    }()
+    private let playButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(" Play", for: .normal)
+        button.setImage(UIImage(systemName: "play.fill"),
+                        for: .normal)
+        button.tintColor = .black
+        button.backgroundColor = .white
+        button.setTitleColor(.black, for: .normal)
+        button.layer.cornerRadius = 3
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 22, weight: .bold)
+        return button
+    }()
+    private let likeButton: UIVerticalButton = {
+        let button = UIVerticalButton()
+        button.setTitle("Like", for: .normal)
+        button.setImage(UIImage(systemName: "heart"),
+                        for: .normal)
+        button.tintColor = .white
+        button.setTitleColor(.white, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        return button
+    }()
+    private var gradientView: UIView = {
+        let gradient = UIView()
+        return gradient
+    }()
```

Comment:
```suggestion
    private var gradientView = UIView()
```
It's better to delete code you don't need 😇